### PR TITLE
PR 2 

### PR DESCRIPTION
**Pull request by CamHargrave**

### CODE REVIEW

File: Adventure/Assets/Scripts/InteractiveObject.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Comment (PatrickIckes) [inline Adventure/Assets/Scripts/InteractiveObject.cs:4]: Add a summary here so someone can understand what this script does.

File: Adventure/Assets/Scripts/RayCast.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Comment (PatrickIckes) [inline Adventure/Assets/Scripts/RayCast.cs:5]: Again, add a summary.

File: Adventure/Assets/Scripts/RayCast.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+public class RayCast : MonoBehaviour
+{
+    [Tooltip("Starting point f raycast used to detect interatives.")]
```
Comment (PatrickIckes) [inline Adventure/Assets/Scripts/RayCast.cs]: Spelling mistake - "interactives"

Comment (PatrickIckes) [reply Adventure/Assets/Scripts/RayCast.cs]: After making all the changes, you're all good for merging.

**Comment by dantogno** [comment]
@CamHargrave Make these changes so @PatrickIckes can merge your branch!!

File: Adventure/Assets/Scripts/InteractiveObject.cs (hunk 1 above)
Comment (CamHargrave) [reply Adventure/Assets/Scripts/InteractiveObject.cs:4]: Added the summary

File: Adventure/Assets/Scripts/RayCast.cs (hunk 2 above)
Comment (CamHargrave) [reply Adventure/Assets/Scripts/RayCast.cs:5]: Added summary

File: Adventure/Assets/Scripts/RayCast.cs (hunk 3 above)
Comment (CamHargrave) [reply Adventure/Assets/Scripts/RayCast.cs]: fixed the spelling mistake

**Pull request merged**